adoc/bp_chap_troubleshooting.adoc

```diff
@@ -312,6 +312,72 @@ For more information see also:
 ** https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=3444
 
 
+[[bp.chapt.suma3.troubleshooting.spacewalk-report]]
+== Gathering Information with [command]``spacewalk-report``
+
+There are instances where administrators may need a concise, formatted summary of their {susemgr} resources, whether it is to take inventory of their entitlements, subscribed systems, or users and organizations.
+Rather than gathering such information manually from the {susemgr} {webui}, {susemgr} includes the [command]``spacewalk-report`` command to fetch and display vital {susemgr} information at once.
```

Comment:
You've buried the lede ;) 
Try something like this:

`The [command]``spacewalk-report`` command is used to produce a variety of reports for system administrators.
These reports can be helpful for taking inventory of your entitlements, subscribed systems, users, and organizations.
Using reports is often simpler than gathering information manually from the {susemgr} {webui}, especially if you have many systems under management.`

Reply:
Yes, I usually do ;)  In this case, it was literally copied from the 2.1 guide. I know this does not make it any better...